.gitlab/pipeline/zombienet/bridges.yml

```diff
@@ -47,8 +47,6 @@
     - cp -r /tmp/bridges-tests-run-*/bridge_hub_rococo_local_network/*.log ./zombienet-logs/
     # copy logs of westend nodes
     - cp -r /tmp/bridges-tests-run-*/bridge_hub_westend_local_network/*.log ./zombienet-logs/
-  tags:
```

Comment:
Ah, sorry about the noice. I set the tag for debugging in the zombienet cluster.

Reply:
Oh, ok. I see that without the tag it advances a bit further. :D Will remove it, but still have to investigate one more issue